applications/vanilla/js/autosave.js

```diff
@@ -6,12 +6,19 @@ jQuery(document).ready(function($) {
         if (!options.button)
             return false;
 
-        var lastVal = null;
+        var lastVal = $(textarea).val();
 
         var save = function() {
             var currentVal = $(textarea).val();
-            if (currentVal != undefined && currentVal != '' && currentVal != lastVal) {
-                lastVal = currentVal
+            var defaultValues = [
+                undefined,
+                null,
+                '',
+                '[{"insert":"\n"}]',
+                lastVal
+            ];
+            if (!defaultValues.includes(currentVal)) {
```

Comment:
Is `includes` supported by Internet Explorer 11?

Reply:
It's polyfilled by `babel-polyfill`. https://github.com/vanilla/vanilla/blob/master/build/entries/polyfills.ts#L11